Editor/Data/Type/CollapsibleUnityEventDrawer.cs

```diff
@@ -0,0 +1,103 @@
+﻿namespace Zinnia.Data.Type
+{
+    using System.Collections;
+    using System.Linq;
+    using System.Reflection;
+    using UnityEditor;
+    using UnityEditorInternal;
+    using UnityEngine;
+    using UnityEngine.Events;
+    using UnityEngine.EventSystems;
+
+    [CustomPropertyDrawer(typeof(UnityEventBase), true)]
+    [CustomPropertyDrawer(typeof(UnityEvent), true)]
+    [CustomPropertyDrawer(typeof(UnityEvent<>), true)]
+    [CustomPropertyDrawer(typeof(UnityEvent<BaseEventData>), true)]
+    internal class CollapsibleUnityEventDrawer : UnityEventDrawer
+    {
+        protected const float headerHeight = 20f;
+        protected const float padding = 6f;
+        protected const float heightOffset = 2f;
+        protected readonly GUIStyle headerBackground = "RL Header";
```

Comment:
```suggestion
        protected const GUIStyle headerBackground = "RL Header";
```
If that doesn't work I'd suggest this instead:
```suggestion
        protected static readonly GUIStyle headerBackground = "RL Header";
```

Reply:
This in fact should be 
`protected static readonly GUIStyle headerBackground = new GUIStyle("RL Header");`

I think if you don't creata a new GUIStyle then it just references the current session style in the Unity editor and if you change anything you end up changing the style editorwide.

Reply:
actually making it static breaks things too.... shruggie emoji

Reply:
Leaving as is then haha 🤷‍♂